packages/client/src/seams/types.ts

```
@@ -89,13 +89,21 @@ export interface HttpResponseData {
   body: Uint8Array;
 }
 
+export type CappedHttpResult =
+  | ({ kind: 'response' } & HttpResponseData)
+  | { kind: 'tooLarge'; observed: number; limit: number };
```

ESLint

[error] Replace `|·({·kind:·'response'·}·&·HttpResponseData)⏎·` with `({·kind:·'response'·}·&·HttpResponseData)`

(prettier/prettier)